oadr/oadr/manager/VENManager.cpp

```diff
@@ -338,11 +353,7 @@ void VENManager::run()
 
 void VENManager::stop()
 {
-	std::unique_lock<std::mutex> lock(m_mutex);
```

Comment:
I believe that this lock provides no real value. The field m_shutdown is used internally and in some cases the process could hang when trying to stop it.